policy/leaving-governance.md

```diff
@@ -4,13 +4,24 @@ This documents the process of leaving Electron Governance. A contributor is no l
 
 ## Offboarding Items
 
-* Update README file for the Working Group. Typically the current WG chair will submit this, but any member of the WG can submit.
+1. A governance member (Typically the WG Chair) update README file in [governance](https://github.com/electron/governance) for the WG the member is leaving.
 
-* Submit a PR in [.permissions repo](https://github.com/electron/.permissions/). Typically the current WG chair will submit this, but any member of the WG can submit.
-    * Merging this PR will automatically remove the contributor from:
-        * The WG's team on GitHub
-        * The WG's user group on Slack
+2. A governance member (Typically the WG Chair) submits a PR in [.permissions repo](https://github.com/electron/.permissions/).
```

Comment:
- copypaste capitalization error from previous sentence
- "in .permissions repo" grammar error
- doesn't document what change is actually to be made in the PR. Is removing the username from the WG sufficient?